frontend/packages/noobaa-storage-plugin/src/components/object-bucket-page/create-ob.tsx

```diff
@@ -0,0 +1,112 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import { Link } from 'react-router-dom';
+import { ActionGroup, Button } from '@patternfly/react-core';
+import { k8sCreate, K8sResourceKind, referenceFor } from '@console/internal/module/k8s';
+import { ButtonBar, history, resourceObjPath } from '@console/internal/components/utils';
+import { StorageClassDropdown } from '@console/internal/components/utils/storage-class-dropdown';
+import { NooBaaObjectBucketModel } from '@console/noobaa-storage-plugin/src/models';
+import { defaultState, commonReducer } from './state';
+
+export const CreateOBPage: React.FC = () => {
+  const [state, dispatch] = React.useReducer(commonReducer, defaultState);
+
+  React.useEffect(() => {
+    const obj: K8sResourceKind = {
+      apiVersion: `${NooBaaObjectBucketModel.apiGroup}/${NooBaaObjectBucketModel.apiVersion}`,
+      kind: NooBaaObjectBucketModel.kind,
+      metadata: {
+        name: state.name,
+      },
+      spec: {
+        ssl: false,
+      },
+    };
+    if (state.scName) {
+      obj.spec.storageClassName = state.scName;
+    }
+    dispatch({ type: 'setPayload', payload: obj });
+  }, [state.name, state.scName]);
+
+  const save = (e: React.FormEvent<EventTarget>) => {
+    e.preventDefault();
+    dispatch({ type: 'setProgress' });
+    k8sCreate(NooBaaObjectBucketModel, state.payload)
+      .then((resource) => {
+        dispatch({ type: 'unsetProgress' });
+        history.push(resourceObjPath(resource, referenceFor(resource)));
+      })
+      .catch((err) => {
+        dispatch({ type: 'setError', message: err.message });
+        dispatch({ type: 'unsetProgress' });
+      });
+  };
+
+  return (
+    <div className="co-m-pane__body co-m-pane__form">
+      <Helmet>
+        <title>Create Object Bucket</title>
+      </Helmet>
+      <h1 className="co-m-pane__heading co-m-pane__heading--baseline">
+        <div className="co-m-pane__name">Create Object Bucket</div>
+        <div className="co-m-pane__heading-link">
+          <Link to="/k8s/cluster/objectbucket/~new" id="yaml-link" replace>
```

Comment:
Remove id `yaml-link` as its not being used anywhere.

Reply:
ack